freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java

```diff
@@ -159,11 +165,34 @@ public void userSetPassword(String user, String password) throws FreeIpaClientEx
         // FreeIPA expires any password that is set by another user. Work around this by
         // performing a separate API call to set the password expiration into the future
         userMod(user, "userpassword", password);
-        updateUserPasswordExpiration(user);
+        updateUserPasswordMaxExpiration(user);
+    }
+
+    public void updateUserPasswordExpiration(String user, long expiration) throws FreeIpaClientException {
+        String passwordExpirationDate = formatDate(expiration);
+        userMod(user, "setattr", "krbPasswordExpiration=" + passwordExpirationDate);
+    }
+
+    public void updateUserPasswordMaxExpiration(String user) throws FreeIpaClientException {
+        updateUserPasswordExpiration(user, 0);
```

Comment:
same question. Here you pass 0, does that mean that the password never expires? Also, maybe have a readability constant for this?

Reply:
I'll add a constant